mt-challenger/cmd/mt-challenger/main.go

```diff
@@ -2,12 +2,15 @@ package main
 
 import (
 	"fmt"
+	"os"
+
+	"github.com/urfave/cli"
+
```

Comment:
should delete this line

Reply:
This conforms to the specification of our go format imports